pyannote/audio/core/task.py

```diff
@@ -257,7 +275,7 @@ def train__len__(self):
 
     def collate_fn(self, batch):
         collated_batch = default_collate(batch)
-        if self.augmentation is not None:
+        if self.augmentation is not None and not self.gpu_transforms:
```

Comment:
This would have to be renamed once we switch to `in model` vs. `in dataloader` convention.